scim2-sdk-common/src/main/java/com/unboundid/scim2/common/utils/DateTimeUtils.java

```diff
@@ -135,62 +132,7 @@ public static Calendar parse(@NotNull final String dateStr)
         ? GMT : TimeZone.getTimeZone(parsedTime.getOffset());
 
     Calendar calendar = Calendar.getInstance(zone);
-    setGregorianCalendar(calendar);
     calendar.setTime(timestamp);
     return calendar;
   }
-
-  private static void setGregorianCalendar(@NotNull final Calendar calendar)
-  {
-    if (calendar instanceof GregorianCalendar gregorian)
-    {
-      // Observe a pure Gregorian calendar (i.e., not a Julian calendar) as
-      // Jakarta-RS did.
-      gregorian.setGregorianChange(new Date(Long.MIN_VALUE));
```

Comment:
This is the one change where I don't think we really need to have full equivalency with the objects we return. I'm not sure why JAX-B chose a full Julian calendar, but it feels like something we do not need to continue keeping.

Also, the comment has a typo and should say JAX-B and not Jakarta-RS, but I didn't bother changing that since it's getting deleted anyway.